pytensor/tensor/math.py

```diff
@@ -1092,7 +1092,7 @@ def log1p(a):
 
 
 @scalar_elemwise
-def sgn(a):
+def sign(a):
     """sign of a"""
 
 
```

Comment:
Do you mind adding a compatible `sgn` like this (and make it accessible from `pytensor.tensor` as before.

```suggestion
def sgn(a):
    warnings.warn(
        "sgn is deprecated and will stop working in the future, use sign instead.", 
        FutureWarning,
    )
    return sign(a)
```

Reply:
Hi @ricardoV94 I have implemented the suggested changes, could you please check if we need anything else in this PR?